client/src/client-only-routes/ShowCertification.js

```diff
@@ -198,20 +199,13 @@ class ShowCertification extends Component {
 
     return (
       <div className='certificate-outer-wrapper'>
-        {conditionalDonationSection}
+        <Grid>{conditionalDonationSection}</Grid>
```

Comment:
Same point as `Intro`, this already has a `Grid` element.  How about replacing the inner `Grid` with a `div`?

I guess this is *probably* fine as is, I'm just not sure if there's a side-effect of nesting containers that I just haven't noticed.